src/main/java/uk/gov/pay/products/exception/FailToRotateApiTokenException.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.pay.products.exception;
+
+public class FailToRotateApiTokenException extends RuntimeException {
```

Comment:
I’m being a bit picky here but maybe `FailToReplaceApiTokenException`? To me, “rotate” implies the old token is also revoked.

Reply:
Done.